deployment/7_upgrade_ens.js

```diff
@@ -0,0 +1,31 @@
+const TestENSRegistry = require('../build/TestENSRegistry');
+const TestReverseRegistrar = require('../build/TestReverseRegistrar');
+const ENSResolver = require('../build/ENSResolver');
+const ENSManager = require('../build/ENSManager');
+
+const deploy = async (network) => {
+    const config = configurator.config;
+    const domain = config.ENS.domain;
+    const deploymentAccount = await deployer.signer.getAddress();
+
+	// ENS domain
+	const parts = domain.split('.');
+	const extension = parts[1];
+	const domainName = parts[0];
+
+    // Instantiate the ENS Registry
+    const ensRegistry = await deployer.wrapDeployedContract(TestENSRegistry, config.ENS.ensRegistry);
+
+	// Create the 'argentx.xyz' wallet ENS namespace
```

Comment:
This step is not necessary since we are using the official ENSRegistry. We need to claim ownership of `argent.xyz` for the new ENSManager using the Registrar for .xyz domains (links to DNS record owner).